examples/contacts/data/e19316b0-af33-4c1c-88e3-9ddd5b03c239.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This file was in the .xar deployed to the public repo, so I added it to the version in source control, assuming it was intended to be in the released version.